src/react_native/utils.cljs

```diff
@@ -37,3 +39,18 @@
         props       (when (map? first-child) first-child)
         children    (if props (rest argv) argv)]
     [props children]))
+
+(defn- convert-prop-value
```

Comment:
A few unit tests for this function would be nice given its importance.

Reply:
Already added some tests :+1: